doc/api/assert.md

```diff
@@ -1074,6 +1074,12 @@ assert.strictEqual('Hello foobar', 'Hello World!');
 // + 'Hello foobar'
 // - 'Hello World!'
 //          ^
+
+assert.strictEqual(1, 2, 'The input values are not equal');
```

Comment:
This causes the linter to fail. We have a lint rule warning against using a literal for the third argument of `assert.strictEqual` because it conceals the values. You can fix it like this perhaps:

```suggestion
const apples = 1;
const oranges = 2;
assert.strictEqual(apples, oranges, `apples ${apples} !== oranges ${oranges}`);
// AssertionError [ERR_ASSERTION]: apples 1 !== oranges 2
```